addons/web/static/src/views/fields/statusbar/statusbar_field.xml

```diff
@@ -13,7 +13,7 @@
     </t>
 
     <t t-name="web.StatusBarField">
-        <div class="o_statusbar_status" t-ref="root" role="radiogroup">
+        <div t-att-class="'o_statusbar_status' + (props.isDisabled ? ' pe-none' : '')" t-ref="root" role="radiogroup">
```

Comment:
The dropdown button should also be disabled. I managed to open the dropdown with keynav